test/TooltipSpec.js

```diff
@@ -0,0 +1,56 @@
+/* global describe, it */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { assert } from 'chai';
+import Tooltip from '../src/Tooltip';
+
+function setup (props = {}) {
+  const propsIn = {
+    ...props
+  };
+  const component = <Tooltip {...propsIn}><button className='btn'>test</button></Tooltip>;
+  const wrapper = shallow(component);
+  const propsOut = wrapper.props();
+  return {
+    propsIn,
+    wrapper,
+    propsOut
+  };
+}
+
+describe('Tooltip', () => {
+  it('should contain class tooltipped', () => {
+    const { wrapper } = setup({ tooltip: '123' });
+    assert.ok(wrapper.hasClass('tooltipped'));
+  });
+
+  it('should convert into child node', () => {
+    const { wrapper } = setup({ tooltip: '123' });
+    assert.ok(wrapper.find('button').length);
```

Comment:
To specifically see if the wrapper is a button, we could assert like so:
`assert.equal(wrapper.type(), 'button');`